internal/github/client.go

```diff
@@ -107,5 +107,5 @@ func (c Client) UploadAsset(ctx context.Context, releaseId int64, release Releas
 
 	opts := &github.UploadOptions{Name: release.AssetPath}
 	asset, _, err := c.gh.Repositories.UploadReleaseAsset(ctx, release.Owner, release.Repo, releaseId, opts, f)
-	return asset.GetBrowserDownloadURL(), err
+	return asset.GetURL(), nil
```

Comment:
pls see the comment in the conversation tab, also we are just returning nil here even if there is error coming from github.